frontend/cypress/tests/chartView/chartView_mergeGroup.cy.js

```diff
@@ -26,9 +28,12 @@ describe('merge group', () => {
     cy.get('div.mui-select.grouping > select:visible')
       .select('groupByAuthors');
 
+    cy.wait(200);
```

Comment:
Just curious, is there any more robust way to make this compatible with asynchronous calls as setting a specific value for the waiting time may cause a stability issue here

Reply:
Yeah I would like to avoid doing it this way as well, I am trying to look for better alternatives as I feel that this could break if the browser is slower and takes more than 200ms to refresh. I am open to any suggestions if anyone has any.